<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculationDialogClass</class>
 <widget class="QDialog" name="CalculationDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculationDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select fuzzy control language file :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmb_fcl_files"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Input data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tbl_inputs">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Output data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tbl_outputs">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_eval">
       <property name="text">
        <string>Evaluate</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_exit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cmb_fcl_files</sender>
   <signal>activated(QString)</signal>
   <receiver>CalculationDialogClass</receiver>
   <slot>initFuzzyModel(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_eval</sender>
   <signal>clicked()</signal>
   <receiver>CalculationDialogClass</receiver>
   <slot>evaluate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_exit</sender>
   <signal>clicked()</signal>
   <receiver>CalculationDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbl_inputs</sender>
   <signal>cellChanged(int,int)</signal>
   <receiver>CalculationDialogClass</receiver>
   <slot>changeInputData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>initFuzzyModel(QString)</slot>
  <slot>changeInputData()</slot>
  <slot>evaluate()</slot>
 </slots>
</ui>
